setup.cfg

```diff
@@ -73,6 +73,7 @@ ignore =
 per-file-ignores =
     # need to import things to top level even if they aren't used there
     desc/*/__init__.py: F401
+    desc/backend.py: F401
```

Comment:
Why is this needed?

Reply:
For some reason, flake was complaining about imports not being used.